README.md

```diff
@@ -90,13 +101,90 @@ docker-compose exec connect kafka-console-consumer --topic pageviews --bootstrap
 
 # Configuration
 
+## Generic Kafka Connect Parameters
+
+See all Kafka Connect [configuration parameters](https://docs.confluent.io/current/connect/managing/configuring.html).
+
+## Connector-specific Parameters
+
 See `kafka-connect-datagen` [configuration parameters](https://github.com/confluentinc/kafka-connect-datagen/blob/master/src/main/java/io/confluent/kafka/connect/datagen/DatagenConnectorConfig.java) and their defaults.
 
-# Schemas for Random Data
+## Use a bundled schema specifications
+
+There are a few quickstart schema specifications bundled with `kafka-connect-datagen`, and they are listed in this [directory](https://github.com/confluentinc/kafka-connect-datagen/tree/master/src/main/resources).
+To use one of these bundled schema, refer to [this mapping](https://github.com/confluentinc/kafka-connect-datagen/blob/master/src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java#L66-L73) and in the configuration file, set the parameter `quickstart` to the associated name.
+For example:
+
+```bash
+...
+"quickstart": "users",
+...
+```
+
+## Define a new schema specification
+
+You can also define your own schema specifications if you want to customize the fields and their values to be more domain specific or to match what your application is expecting.
+Under the hood, `kafka-connect-datagen` uses [Avro Random Generator](https://github.com/confluentinc/avro-random-generator), so the only constraint in writing your own schema specification is that it is compatible with Avro Random Generator.
+To define your own schema:
+
+1. Create your own schema file `/path/to/your_schema.avro` that is compatible with [Avro Random Generator](https://github.com/confluentinc/avro-random-generator)
```

Comment:
You should use the `.avsc` file extension for schema definition files not `.avro`

Reply:
Ack and done